src/apiCalls.js

```diff
@@ -1,8 +1,17 @@
 // Fetch requests
+const throwError = (response) => {
```

Comment:
Excellent user of leveraging the condition of the response to perform the correct action!